### ISSUE TITLE
Update ApiHub trigger functionality to support retry in cases when the trigger notification is received but before the user function is called.

### ISSUE THREAD

**Issue by safihamid** [Azure/azure-webjobs-sdk-extensions#99]

**Comment by mathewc** [comment]
But this retry logic would have to be in the ApiHub service itself, not in this extension, right? E.g. if the WebJob receives the trigger, but before user function is called **the VM crashes**. Retry logic will have to be outside.

**Comment by christopheranderson** [comment]
@safihamid - is this still active?

**Comment by safihamid** [comment]
Yes this is what we are discussing to make design changes either in WebJobs SDK or in the extensions repo.
